scss/components/_headings.scss

```diff
@@ -0,0 +1,124 @@
+// Headings
+//
+// Contain the title, date select and action button.
+// Use .heading--main when the heading is the top-level heading on a page as it provides extra margin-bottom
+// Use .heading--section When the heading is just a heading of a section on a page for less margin-bottom
```

Comment:
Just figured out that we can use markdown formatting here to format, so I used the tic marks.